### PR TITLE
hardening/36_add_config_section

### PR DESCRIPTION
**Pull request by frbattid**
- Implements issue #36
- No unit tests nor e2e test required.
- Assignee @fgalan

### CODE REVIEW

File: tidoop-mr-lib-api/README.md [hunk 1]
```diff
@@ -73,10 +88,10 @@ The Http server implemented by tidoop-mr-lib-api is run as (assuming your curren
     
 If everything goes well, you should be able to remotely ask (using a web browser or `curl` tool) for the version of the software:
 
-    $ curl -X GET "http://<host_running_the_api>:6060/version"
+    $ curl -X GET "http://<host_running_the_api>:12000/version"
```
Comment (fgalan) [inline tidoop-mr-lib-api/README.md:91]: Just curioristy... why the port has changed?

Comment (frbattid) [inline tidoop-mr-lib-api/README.md:91]: Well:
- Hive port is 10000
- Oozie port is 11000
- OAuth2 tokens generator port is 13000
- HttpFS port is 14000

As you can see, 12000 is free; and 6060 has not any relation with the above ones... Look and feel business :)

Comment (fgalan) [inline tidoop-mr-lib-api/README.md:91]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**